.travis.yml

```diff
@@ -18,91 +18,22 @@ stages:
 - build
 matrix:
   include:
-  - name: webpack build
-    language: python
-    python:
-    - '2.7'
-    stage: build
-    install:
-    - pip install -r requirements.txt
-    - find designsafe/settings/* -name *.sample.* | while read line; do cp $line ${line/sample\./};
-      done;
-    - "./manage.py collectstatic --settings=designsafe.settings.test_settings --no-input
-      --link -i demo > /dev/null"
-    - npm install
-    script:
-    - npm run build
-  - name: npm test
-    language: python
-    python:
-    - '2.7'
-    stage: test
-    install:
-    - pip install -r requirements.txt
-    - find designsafe/settings/* -name *.sample.* | while read line; do cp $line ${line/sample\./};
-      done;
-    - "./manage.py collectstatic --settings=designsafe.settings.test_settings --no-input
-      --link -i demo > /dev/null"
-    - npm install
-    - npm install -g codecov
-    script:
-    - npm run test
-    after_script:
-    - npm run report-coverage
-  - name: unittests
-    language: python
-    python:
-    - '2.7'
-    stage: test
-    install:
-    - pip install -r requirements.txt
-    - find designsafe/settings/* -name *.sample.* | while read line; do cp $line ${line/sample\./};
-      done;
-    - "./manage.py collectstatic --settings=designsafe.settings.test_settings --no-input
-      --link -i demo > /dev/null"
-    script:
-    - coverage run --branch --source='designsafe' manage.py test --settings=designsafe.settings.test_settings
-      --pattern=tests.py -v2
-    after_script:
-    - coverage xml
-    - codecov
   - name: Flake8
     language: python
     python:
-    - '2.7'
+    - '3.6'
     stage: lint
     install:
-    - pip install -r requirements.txt
-    - find designsafe/settings/* -name *.sample.* | while read line; do cp $line ${line/sample\./};
-      done;
-    - "./manage.py collectstatic --settings=designsafe.settings.test_settings --no-input
-      --link -i demo > /dev/null"
+    - pip3 install -r requirements.txt
```

Comment:
this makes it so that pip3 is explicitly used to install requirements like it is when building docker images for prod